sentences/en/light_HassLightSet.yaml

```diff
@@ -71,6 +65,23 @@ intents:
           area:
             slot: true
 
+      # Floor support for brightness
+      - sentences:
+          - "[<numeric_value_set>] [the] brightness in <floor> to <brightness>"
+          - "[<numeric_value_set>] [the] brightness of <floor> to <brightness>"
+          - "[<numeric_value_set>] <floor> brightness [to] <brightness>"
+          - "[<numeric_value_set>] <floor> [to] <brightness> brightness"
+          - "[<numeric_value_set>] <floor> [to] <brightness>"
```

Comment:
This will surely clash with other sentences in the future. Or with volume at the moment. We should specifically include `brightness` in the sentence.

Reply:
It's already the case for area too.
The sentence exists without `brightness`

Reply:
Same thing, it was made that way when there was nothing other than brightness :) That doesn't mean it's still OK, but leave it as you see fit and I'll try to update when I can (for both floors and areas).

Reply:
I am simplifying sentences. The main idea was to make sure something was present in the english language, but you are correct, it's technically not my job to do so.
So I will only put a single sentence.

Reply:
i.e. I'm trying to block as little creative effort as possible and at the same time to create as few future problems as possible.